addons/event/controllers/main.py

```diff
@@ -23,3 +26,56 @@ def event_ics_file(self, event, **kwargs):
             ('Content-Length', len(content)),
             ('Content-Disposition', content_disposition('%s.ics' % event.name))
         ])
+
+    @route(['/event/my_tickets/<model("event.event"):event>'], type='http', auth='public')
```

Comment:
good catch about this one, maybe it should be an integer and we sudo it below (double check with the PO to be sure, but if the user has the hash, he should be able to bypass acl imo)

Reply:
Yes, we all agree on the sudo :) Done